.github/workflows/main.yml

```diff
@@ -17,6 +17,11 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
+      - name: Use Node 23
+        uses: actions/setup-node@v4
+        with:
+          node-version: 23
```

Comment:
don't use odd versions of Node because their lifespan is really short. Switch to Node 24 or 22 instead